### PR TITLE
BaSP-MR: Table Trainers

### PR DESCRIPTION
**Pull request by Emanuel-Lamberti**
Modifications were implemented in the Table component in Trainers, after adding Redux to it in its lines of code.
GET and DELETE methods were used in this branch.

### CODE REVIEW

File: src/redux/trainers/thunks.js [hunk 1]
```diff
@@ -32,3 +38,41 @@ export const updateTrainer = async (dispatch, id, body) => {
...
+
+export const getTrainers = async (dispatch) => {
+  try {
+    const reponse = await fetch(`${process.env.REACT_APP_API_URL}/api/trainer`);
+    const data = await reponse.json();
+    const listTrainers = data.data;
+    console.log(listTrainers);
+    if (listTrainers.length === 0) {
+      dispatch(getTrainersPending(true));
+      console.log(true);
```
Comment (IvanJukonis) [inline src/redux/trainers/thunks.js:50]: This console.log should be deleted.

**Review by IvanJukonis — CHANGES_REQUESTED** [review]
Nice work Ema! I pointed out some console.logs that I think should be deleted.

File: src/redux/trainers/thunks.js (hunk 1 above)
Comment (Emanuel-Lamberti) [reply src/redux/trainers/thunks.js:50]: I'm working in it. Thanks for notifying.

File: src/redux/trainers/thunks.js [hunk 2]
```diff
@@ -32,3 +38,41 @@ export const updateTrainer = async (dispatch, id, body) => {
...
+    } else {
+      dispatch(getTrainersPending(false));
+      console.log(false);
```
[the remark this answers is not in the document]
Comment (Emanuel-Lamberti) [reply src/redux/trainers/thunks.js:53]: I'm working in it. Thanks for notifying.

File: src/redux/trainers/thunks.js [hunk 3]
```diff
@@ -32,3 +38,41 @@ export const updateTrainer = async (dispatch, id, body) => {
     dispatch(updateTrainerError(true));
   }
 };
+
+export const getTrainers = async (dispatch) => {
+  try {
+    const reponse = await fetch(`${process.env.REACT_APP_API_URL}/api/trainer`);
+    const data = await reponse.json();
+    const listTrainers = data.data;
+    console.log(listTrainers);
```
[the remark this answers is not in the document]
Comment (Emanuel-Lamberti) [reply src/redux/trainers/thunks.js:47]: I'm working in it. Thanks for notifying.

**Review by CristianLotorto — APPROVED** [review]
Great Job Ema!

File: src/Components/Trainers/index.js [hunk 4]
```diff
@@ -61,6 +44,7 @@ function Trainers() {
           deleteButton={deleteTrainer}
           columns={columnsValue}
           autoDelete={() => {}}
```
Comment (FrancoLelli) [inline src/Components/Trainers/index.js:46]:
```suggestion
  
```

File: src/Components/Trainers/index.js [hunk 5]
```diff
@@ -61,6 +44,7 @@ function Trainers() {
...
+          arrayAndObject={[]}
```
Comment (FrancoLelli) [inline src/Components/Trainers/index.js:47]:
```suggestion

```

**Review by FrancoLelli — CHANGES_REQUESTED** [review]
Delete!!

File: package.json [hunk 6]
```diff
@@ -12,7 +12,6 @@
     "react-router-dom": "^5.3.4",
     "react-scripts": "5.0.1",
     "redux": "^4.2.1",
-    "redux-devtools-extension": "^2.13.9",
     "redux-thunk": "^2.4.2",
     "web-vitals": "^2.1.4"
```
Comment (YhanahiRosenthal) [inline package.json:16]: you should have this dependencie installed!

**Review by IvanJukonis — APPROVED** [review]

**Review by IvanJukonis — APPROVED** [review]

File: src/Components/Trainers/index.js (hunk 4 above)
Comment (Emanuel-Lamberti) [reply src/Components/Trainers/index.js:46]: Done

**Review by FrancoLelli — APPROVED** [review]

**Pull request merged**